.github/workflows/tests.yml

```diff
@@ -41,3 +41,15 @@ jobs:
       shell: bash -l {0}
       run: |
         conda build -q conda-recipe --package-format=1
+    - name: Build and upload sphinx documentation to NNPDF server
```

Comment:
Since we are moving towards a tagged release schedule, we should also provide tagged documentation.

On RTD this is almost trivial as I just found out: https://docs.readthedocs.com/platform/stable/guides/automation-rules.html . And to be fair we haven't done that for, e.g., eko so far - but now we do: https://eko.readthedocs.io/en/v0.14.6/code/Operators.html . Now this version (v0.14.6) I added manually as I think the rules are only from now on. Until now for eko it was only possible to see either the version of `master` or `v0.8` from 3 years ago, which is clearly not ideal - since we know master is not stable and v0.8 is ancient ... also it turned out that I could not add the v0.13.5 version since in that commit RTD is broken :melting_face:

Reply:
Yes, I was thinking doing so, but we are not using RTD but our server because we are too cool for school...

It should be easy to do in the server just the same of course, but right now all tagged docs would be outdated anyway so better to keep updating them until we catch up :__)